easybuild/easyconfigs/b/BEDTools/BEDTools-2.29.2-GCC-8.3.0.eb

```diff
@@ -0,0 +1,42 @@
+# Author: Maxime Schmitt, University of Luxembourg
+# Author: Adam Huffman, The Francis Crick Institute
+#
+# Based on the work of: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics (SIB)
+# Biozentrum - University of Basel
+
+easyblock = 'MakeCp'
+
+name = 'BEDTools'
+version = '2.29.2'
+
+homepage = "https://bedtools.readthedocs.io/"
+description = """BEDTools: a powerful toolset for genome arithmetic.
+The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps and
+computing coverage.
+The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF, and SAM/BAM."""
+
+toolchain = {'name': 'GCC', 'version': '8.3.0'}
+
+source_urls = ['https://github.com/arq5x/bedtools2/releases/download/v%(version)s/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['e3f1bf9e58740e60c3913390fe95b0c7f8fd99ceade8a406e28620448a997054']
+
+buildopts = 'CXX="$CXX"'
+
+dependencies = [
+    ('binutils', '2.32'),
```

Comment:
@Falkor `binutils` is not a runtime dependency for `BEDTools`, is it? Please remove this line.

Reply:
done in 1cf3d18